### DISCUSSION TITLE
How can I use this tool with Argon?

### DISCUSSION THREAD

**Discussion by obvRedwolf** [JohnnyMorganz/luau-lsp#445] [Q&A]
Currently, I don't know how I would use this tool with Argon, as no sourcemap file is generated. Please help!

**Comment by JohnnyMorganz** [comment]
What you need is a `sourcemap.json` file in the same structure as the one output via Rojo.

You can do it in a few ways. You can manually make one or create a script to do so (you will have to manually update it too on any changes). Or argon can add a sourcemap command

**Reply by obvRedwolf** [reply]
I meant how can I make one myself? I'm also using Argon with my friends who are using Roblox Team Create.
